.github/workflows/integration.yml

```diff
@@ -20,6 +20,8 @@
 
 name: Adapter Integration Tests
 
+run-name: "${{ contains(github.event_name, 'workflow_') && inputs.name || github.event_name }}: ${{ contains(github.event_name, 'workflow_') &&  inputs.adapter_branch || github.ref_name }} by @${{ github.actor }}"
```

Comment:
I know how the mechanic `contains() && string` works, but it's not intuitive. Between that and the combination of an `&&` and an `||`, I would put parentheses around the `&&` expressions.

Reply:
Parentheses added.